### DISCUSSION TITLE
View and list PR?

### DISCUSSION THREAD

**Discussion by PMassicotte** [jesseduffield/lazygit#3701] [General]
First, thank you for this great piece of software.

I was wondering if it is possible to check out incoming PRs. Using the `gh` CLI command, I can do `gh pr list` and then `gh pr checkout #`, where `#` is the PR ID.

Is it possible to view PRs within `lazygit`?

**Comment by nivv** [comment]
This would be great since Lazygit lists closed and deleted branches in the remotes list.  I have hundreds now for some projects and it's hard to get the latest one.

**Reply by mark2185** [reply]
> Lazygit lists closed and deleted branches in the remotes list

I have this in my `.gitconfig` to remove non-existing references on the remote:
```
[fetch]
	prune = true
```

**Reply by nivv** [reply]
Thanks, that makes the workflow a lot nicer!
